src/Microsoft.SqlTools.ServiceLayer/DisasterRecovery/BackupFactory.cs

```diff
@@ -7,20 +7,21 @@
 using System.Collections;
 using System.ComponentModel;
 using System.Data.SqlClient;
+using Microsoft.Data.Tools.DataSets;
 using Microsoft.SqlServer.Management.Sdk.Sfc;
 using Microsoft.SqlServer.Management.Smo;
 using Microsoft.SqlServer.Management.Common;
-using Microsoft.SqlTools.ServiceLayer.Common;
-using Microsoft.Data.Tools.DataSets;
+using Microsoft.SqlTools.ServiceLayer.Admin;
 using Microsoft.SqlTools.ServiceLayer.DisasterRecovery.Contracts;
+using System.Collections.Generic;
 
 namespace Microsoft.SqlTools.ServiceLayer.DisasterRecovery
 {
     public class BackupFactory
```

Comment:
>BackupFactory [](start = 17, length = 13)

this class seems more like a helper class than a factory (factories typically just have a static method that creates a object, or something similar).  Could we rename something like BackupHelper or BackupUtilities or something like that?